src/view/com/util/load-latest/LoadLatestBtn.tsx

```diff
@@ -43,7 +45,7 @@ export function LoadLatestBtn({
           (isTallViewport
             ? styles.loadLatestOutOfLine
             : styles.loadLatestInline),
-        isTablet && styles.loadLatestInline,
+        isTablet && !isNativeTablet && styles.loadLatestInline,
```

Comment:
Just curious how often we think this will happen? Could add a `isWebTablet` for this if it will happen a lot. Nbd if this is a rare one-off thing though.

Reply:
That's probably a good idea, yeah